### PR TITLE
Migrated to Sass and changed name to Steps

### PR DESCRIPTION
**Pull request by jakepeyser**
- refactored CSS into Sass files
- changed name of app to Steps
- enabled hot module replacement via `webpack-dev-server` and `npm hmr` command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,19 +8,37 @@ _Physical therapy for the way you live today_
...
+npm run hmr
+```
+
+The command starts a little Node.js server that serves only the webpack bundle. This setup gives us two main advantages:
 
-1. Install npm dependencies
-1. Perform a build with Webpack and watch for changes
-1. Run the server and watch for file changes with nodemon
+- Changes to React components cause automatic page refreshes
+- Updates to static files like CSS are injected into the running app without requiring a full refresh
```
Comment (Hitenp1) [inline README.md]: Would be great if we could go over maybe after scrum tomorrow in what situations to run the commands

Comment (jakepeyser) [inline README.md]: yep, I'll demo during Monday's scrum

File: src/index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <meta content="<%= htmlWebpackPlugin.options.meta[key] %>" name="<%= key %>">
     <% } %>
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <link href="/bootstrap.min.css" rel="stylesheet" type="text/css">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
   </head>
```
Comment (Hitenp1) [inline src/index.html]: Does the bootstrap link you provided limit it to version 3.3.7 whereas the other one is updated to the most recent version? also is this to speed up content loading?

Comment (jakepeyser) [inline src/index.html]: > Does the bootstrap link you provided limit it to version 3.3.7 whereas the other one is updated to the most recent version?

Yes, we will have to manually update the CDN link for version updates, whereas the other method would at least give us automatic patch-level updates. However, this was the easiest way to get the `webpack-dev-server` working. I can template which link we use based on environment if everyone thinks this is important.

> is this to speed up content loading?

That was not the intention, but yes, using a CDN will speed up the load time for that script

**Pull request merged**